e2e-tests/gatsby-image/cypress/integration/fixed.js

```diff
@@ -24,7 +24,7 @@ describe(`fixed`, () => {
 
   it(`applies 1x/2x`, () => {
     cy.getTestElement(fixedTestId)
-      .find(`picture > source`)
+      .find(`picture:nth-of-type(2) > source`)
```

Comment:
The numbers `1` and `2` in all these tests should probably use a variable to represent the target? Or we could extract the common part of the selector to a descriptive variable name which should suffice:

```JS
const placeholderImage = `picture:nth-of-type(2)`
const primaryImage = `picture:nth-of-type(2)`
// ...
.find(`${primaryImage} > source`)
.find(`${primaryImage} > img`)
// ...
.find(`${placeholderImage} > source`)
.find(`${placeholderImage} > img`)
```